lib/afip_bill/generator.rb

```diff
@@ -11,7 +11,7 @@ class Generator
 
     HEADER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_header.html.erb'.freeze
     FOOTER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_footer.html.erb'.freeze
-    BRAVO_CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B" }.freeze
+    CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B" }.freeze
```

Comment:
@pipimumu Please update the name on the constant in the `type_a_or_b_bill` method as well

Reply:
@lubc  I have made  the aforesaid changes. Please do have a look!